scripts/youtube.js

```diff
@@ -244,33 +244,6 @@ H5P.VideoYouTube = (function ($) {
 
     };
 
-    // xAPI extension events for video.
-    self.on('seeked', function (event) {
-      this.triggerXAPI('seeked', event.data);
-    });
-    self.on('volumechange', function (event) {
-      // @todo: Not currently used.
```

Comment:
Not sure if we should keep these todo marks; Kirk put them there as hints about what the specific player did not implement. In this case, the youtube player doesn't fire a volumechange event when the volume is changed.